src/mito2/src/compaction.rs

```diff
@@ -362,9 +362,13 @@ impl CompactionScheduler {
 
         // Submit the compaction task.
         self.scheduler
-            .schedule(Box::pin(async move {
-                local_compaction_task.run().await;
-            }))
+            .schedule(Job::new(
+                "compaction",
+                Priority::Low(Some(Duration::from_secs(5 * 60))),
```

Comment:
out-of-topic: We may assign high-pri or allow configuring the priority if the compaction is a manually triggered by users.

Reply:
> out-of-topic: We may assign high-pri or allow configuring the priority if the compaction is a manually triggered by users.

Good catch. I fixed in b68a73a5d0 so that manually triggered compaction tasks will obtain High priorities.